DEPENDENCIES.md

```diff
@@ -0,0 +1,17 @@
+# Dependencies
+
+### moonlink
+Moonlink is a Rust library that provides real-time synchronization between Postgres and Iceberg. It is a key component enabling pg_mooncake columnstore to support point inserts, updates, and deletes, while maintaining consistency with the base OLTP table.
+
+- **Version**: v0.1.0
```

Comment:
We might need to be consistent with this https://github.com/Mooncake-Labs/moonlink/blob/f61d948c87ff6283513d5a45d9990227862f19a1/Cargo.toml#L6